lib/fhir/resources/value-set.js

```diff
@@ -0,0 +1,47 @@
+/**
+* Copyright (c) 2017-present, Jembi Health Systems NPC.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict'
+
+const constants = require('../../constants')
+const FhirCommon = require('../common')
+const QueryUtils = require('../query-utils')
+
+module.exports = (mongo) => {
+  const fhirCommon = FhirCommon(mongo)
+  const queryUtils = QueryUtils(mongo)
+
+  return {
+    name: 'ValueSet',
+
+    searchFilters: (ctx, callback) => {
+      const queryParams = ctx.query
+      const supportedParams = {
+        url: {}
```

Comment:
I think we should add the "ValueSet.codeSystem.system" as well, as this property defines the system for the code set

Reply:
Oops, I misread the search prams table in the resource :/